src/now.js

```diff
@@ -11,9 +11,9 @@ if (!process.browser) {
     return hr[0] * 1e9 + hr[1]
   }
   const loadTime = getNanoSeconds()
-  nowForNode = () => ((getNanoSeconds() - loadTime) / 1e6)
+  nowPolyfillForNode = () => ((getNanoSeconds() - loadTime) / 1e6)
 }
 
-export default process.browser
-  ? perf && perf.now ? () => perf.now() : () => Date.now()
-  : nowForNode
+export default perf && perf.now
+  ? () => perf.now()
+  : process.browser ? () => Date.now() : nowPolyfillForNode
```

Comment:
Could probably get a microsecond perf boost from doing `const now = perf.now.bind(perf)`, but meh. I don't want to rely on `bind`, or on `now()` working without the proper `this`, etc.